library/src/morpheus-master/sass/modules/tool/forums/_forums.scss

```diff
@@ -259,4 +261,14 @@
 		width: auto;
 		margin-left: 1em;
 	}
+
+	#revise p.anonTopic.checkbox {
+		display: flex;
+		flex-direction: row;
+
+		input[type=checkbox] {
```

Comment:
This whole way of aligning this special case doesn't really match the alignment of the surrounding checkboxes, but I understand what you are trying to fix and achieve here. Until Sakai has a better form layout definition and better form inline messages, this will work as a temporary fix.